zhaquirks/linxura/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Linxura button device."""
+
+from zigpy.quirks import CustomCluster
+import zigpy.zcl.clusters.security
+
+from zhaquirks.const import (
+    BUTTON,
+    BUTTON_1,
+    BUTTON_2,
+    BUTTON_3,
+    BUTTON_4,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_ID,
+    COMMAND_PRESS,
+    PRESS_TYPE,
+    ZHA_SEND_EVENT,
+)
+
+LINXURA = "Linxura"
+STATUS_REPORT = 2
+
+
+PRESS_TYPES = {
+    1: COMMAND_PRESS,
+    2: COMMAND_DOUBLE,
+    3: COMMAND_HOLD,
```

Comment:
Per the previous comment, we could fire these press types to have them match the trigger we provide to HA.
That would allow us to compact the `device_automation_triggers`.
```suggestion
    1: SHORT_PRESS,
    2: DOUBLE_PRESS,
    3: LONG_PRESS,
```